src/cljs/athens/db.cljs

```diff
@@ -489,9 +489,6 @@
 
 (defn count-linked-references-excl-uid
   [title uid]
-  (reduce (fn [current-count ref]
-            (if (= (:block/uid ref) uid)
-              current-count
-              (inc current-count)))
-          0
-          (get-linked-references-by-block title)))
+  (->> (get-linked-references-by-block title)
+       (remove #(= (:block/uid %) uid))
+       count))
```

Comment:
Thank you! This definitely is much more readable and intuitive. Just a small styling suggestion: maybe move line 494 up to 493 so that the `remove` function is together?

Reply:
I don't understand, could you be more specific.
The form in L494 is independent from the one in L493.
remove -> filtered items
count -> number of filtered items

Reply:
Oh yeah NVM, didn't notice the `->>` operation :)